src/content/app/tools/vep/views/vep-form/vep-form-options-section/VepFormOptionsSection.tsx

```diff
@@ -0,0 +1,109 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useAppSelector } from 'src/store';
+
+import { getVepFormInputCommittedFlag } from 'src/content/app/tools/vep/state/vep-form/vepFormSelectors';
+
+import { useVepFormConfigQuery } from 'src/content/app/tools/vep/state/vep-api/vepApiSlice';
+
+import FormSection from 'src/content/app/tools/vep/components/form-section/FormSection';
+import VepFormGeneOptions from './vep-form-gene-options/VepFormGeneOptions';
+
+import styles from './VepFormOptionsSection.module.css';
+
+/**
+ * TODO:
+ * - Show a pseudo-toggle between short variants and structural variants.
+ * - Move Genes & transcripts section to its own directory?
+ * - Display options
+ *
+ * IDEAS:
+ * - Should user's file be stored in IndexedDB before submission?
+ *  - This means, VEP isn't going to be usable in FF private mode
+ *    - But it won't be usable in that mode anyway
```

Comment:
Yes we don't need support private mode for this. Just my opinion on this though.

Reply:
Yes. I should delete this whole comment anyway. 😄

Reply:
Deleted.